src/clj/views/question/question_view.clj

```diff
@@ -0,0 +1,62 @@
+(ns views.question.question-view
+  (:require
+    [hiccup.form :as form]
+    [hiccup2.core :as h]
+    [ring.util.anti-forgery :refer [anti-forgery-field]]))
+
+
+(def question-errors
+  #{:not-assigned-to-question})
+
+
+;; I dont know how this exactly works.
+;; This function and the subsequent usage
+;; in the dispatch-question-type is
+;; sourced from: https://github.com/yokolet/hiccup-samples
+;; TODO: add answer-id to the lablaed radio
+(defn labeled-radio
+  [label]
+  [:label (form/radio-button {:ng-model "question.answer"} "answer" false label)
+   (str label "    ")])
```

Comment:
AFAIK the `:ng-model` attribute is related to the AngularJS Framework, as the samples you mentioned were written for that, so we don't need this attribute here.
And I also don't understand why they used multiple spaces in this sample, as in HTML multiple spaces are always collapsed into one :thinking: 
So I'd suggest to use a span element + a Bootstrap spacing class instead of that:
```suggestion
  [:label (form/radio-button "answer" false label)
   [:span.mx-1 label])
```